### PR TITLE
data: ibrahimq21

### PR DESCRIPTION
**Pull request by ibrahimq21**
data: ibrahimq21

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3248"></a>

### CODE REVIEW

File: data/IbrahimQureshi.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Ibrahim Qureshi",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am an emerging tech enthusiast with a passion for creating innovative solutions using the latest technologies. Whether",
```
Comment (AbbeyIT) [inline data/IbrahimQureshi.json:5]: Does the bio really ends with "Whether"? It seems like there's more to it

**Comment by ibrahimq21** [comment]
no this "Whether" should be removed

**Comment by AbbeyIT** [comment]
Feel free to edit it @ibrahimq21

**Comment by ibrahimq21** [comment]
editing done

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

Please fix your JSON profile file naming. It should be `{your GitHub username}.json` -> `
ibrahimq21.json`. Otherwise, features will not be functional.

If you need any help feel free to ask.

**Comment by ibrahimq21** [comment]
the username is ibrahimq21 in github is there some other problem let me
know.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**